contracts/abstracted_account.algo.ts

```diff
@@ -173,34 +304,71 @@ export class AbstractedAccount extends Contract {
    * Temporarily rekey to an approved plugin app address
    *
    * @param plugin The app to rekey to
+   * @param methodOffsets The indices of the methods being used in the group
```

Comment:
Honestly I'm still kinda confused on how `methodOffsets` is used

Reply:
answered up above